frontend/components/Logo.tsx

```diff
@@ -3,21 +3,20 @@ export function Logo(props: { class?: string }) {
   return (
     <svg
       xmlns="http://www.w3.org/2000/svg"
-      viewBox="0 0 638 343"
-      fill="none"
+      viewBox="-1.5 -0.5 16 8"
```

Comment:
Nudge the `viewBox` as the artwork fits in 13×7, but a 2:1 aspect ratio that lines up with `rem` is more flexible

Reply:
I have concerns about the viewbox extending beyond the artwork bounds, just because it will mean any time the SVG is used anywhere, there will be negative margins or positioning necessary to align it. Is a way to accomplish this without adding the space?

Reply:
Yes, but for the height, there are not many multiple of 7 within the Tailwind dimensions, hence why 8 units high was chosen. One possibility would be to keep the height at 8 with one empty unit at the bottom rather than 0.5 at the top and bottom.

For the width, it’s entirely possible to keep it at 13 units if you prefer, and I’ll make the change.

Reply:
I've now made the change. How do you suggest we adjust the height? Adding the `h-8` class will not work as that would result in 32px high, which is not divisible by 7.

Here are some options I thought of:

- add a `height` prop with some JSDoc suggesting to use a multiple of 7
- add a `size` prop that takes integers that are multiplied by 7 to produce de final height in pixels
- add a `size` prop that takes a series of values like S M L XL that uses CSS calc to get a valid size in `rem`: `calc(2 * 0.875rem)` for `h-8` equivalent
- do not allow size variations for this component

Reply:
I like the idea of taking a size prop, and making sure all the sizes match up to exact pixel alignments.